build/configs/qemu/qemu-2.10.0-rc2_16m_ram_size.patch

```diff
@@ -1,7 +1,7 @@
-From a9f770243ccb354e99b9ff33ecfecbeca73b1b03 Mon Sep 17 00:00:00 2001
+From 23caf033c718dd932efd225a01d00587b88b081a Mon Sep 17 00:00:00 2001
```

Comment:
Is this file needed??????

Reply:
yes. Currently flash is made as readonly(ROM) inside qemu. 
Hence i have updated that patch to support flash as "read/write" inside qemu. 
Without that smartfs wont work